github/src/main/kotlin/gropius/sync/github/model/LabelInfo.kt

```diff
@@ -1,11 +1,14 @@
 package gropius.sync.github.model
+
 import org.bson.types.ObjectId
 import org.springframework.data.annotation.Id
 import org.springframework.data.mongodb.core.index.Indexed
 import org.springframework.data.mongodb.core.mapping.Document
+import java.net.URI
 
 /**
  * Mapping of a single label from neo4j to github
+ * @param url API URL of IMS of the repo
```

Comment:
consistent documentation in same class

Reply:
f22dc76eecf06c31cba4638560eb8dafa04cf6c7